manim/utils/cfg_file_utils.py

```diff
@@ -1,28 +1,30 @@
 """
-cfgwriter.py
+cfg_file_utils.py
```

Comment:
Comment: I agree this file should be changed, but I'd rather something more descriptive, that says "this is the file that is ran when the cfg subcommand is executed". It is also hard to distinguish between this and `config.py`. Something like `cfg_subcommand.py` for this file, though that's too long... [I'm rambling here...]

Reply:
It also clashes with `config_utils.py`...

Reply:
I've renamed it to `cfg_subcmds.py`. It's basically what you suggested, but who needs vowels anyway :P